README.md

```diff
@@ -15,16 +15,26 @@ This is particularly useful when:
 
 ## The Four Risk Levels
 
-| Risk Level | Code | Example | Meaning |
-| --- | --- | --- | --- |
-| **Known safe** | lowercase letter | `r - Extract method Applesauce` | Addresses all known and unknown risks. |
-| **Validated** | uppercase letter | `R - Extract method Applesauce` | Addresses all known risks. |
-| **Risky** | uppercase followed by 2 bangs | `R!! Extract method Applesauce` | Some known risks remain unverified. |
-| **(Probably) Broken** | uppercase followed by 2 stars | `R** Start extracting method with no name` | No risk attestation. |
-
-* **Known safe:** Developer performed the task in a way that prevents the potential risks, even for situations that developer is not aware of.
-* **Validated:** Developer performed the task in some way that includes validation for all risks the developer thought of. The most common technique is developer-written automated tests.
-* **Risky:** Developer is aware of risks and attempted to mitigate them as much as possible, but there is no formal verification. Commonly this includes a manual change that the developer could not fully verify.
+We divide all behaviors of the system into 3 sets. The change is intended to alter the *Intended Change* while not altering any of the *Invariants*. The *Risk Levels* are based on correctness guarantees: which invariants can this commit guarantee did not change, and can this commit guarantee that it changed the intended change in the way the authors intended?
+
+| Risk Level | Code | Example | Meaning | Correctness Guarantees |
+| --- | --- | --- | --- | --- |
+| **Known safe** | `.` | `. r Extract method` | Addresses all known and unknown risks. | Intended Change, Known Invariants, Unknown Invariants |
```

Comment:
Suggest: Parallelism -- "(Known) Safe", preserving the strong intent of provably safe (like "(Probably) Broken"), and also allowing reference to unambiguous, single words for each level -- Safe, Validated, Risky, Broken.